pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>com.cybersource</groupId>
     <artifactId>cybersource-sdk-java</artifactId>
-    <version>6.0.2-SNAPSHOT</version>
+    <version>6.1.0</version>
```

Comment:
Keep the version as snapshot, we will use maven release plugin to build next version